release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,10 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 12 July 2024 - Enhancement - Catalog - Infinite scroll [ID_40167]
+
+​On the catalog browse page, when users now scroll to the bottom of the page a new page will load if there is any. (The results are still capped at 50 items per page)
```

Comment:
Can also explain what the loading of the next page looks like.
Not sure if it's that important to end users, as they'll hopefully notice it

"The loading of the next page is indicated by a single row at the bottom with skeleton items"